docs/vyper.md

```diff
@@ -0,0 +1,68 @@
+Vyper Compiler
+==============
+
+Vyper contracts can be compiled in Remix in the **Vyper compiler**.
+
+![](images/a-vyper-pi-basic.png)
+
+The plugin can be activated going to:
+1. the Plugin Manager and clicking on the Vyper compiler's activate button.
+2. the File Explorer, then right clicking on a .vy file and selecting compile in the pop-up menu.
+3. https://remix.ethereum.org/#activate=vyper
+
+## Compiling a Vyper file
+A Vyper file (with a .vy extension) needs to be the active file in the Editor.  To get some example files, click on the **Clone Vyper examples repo** button.  This will clone a repo and put it in a new Workspace.  Go to the File Explorer to see the files.
```

Comment:
`A Vyper file (with a .vy extension) needs to be the active file in the Editor` ---> this seems an incomplete or irrelevant sentence

Reply:
I disagree.  Someone could just activate the plugin and wonder why it isn't working.  This explains how to make the compile button active.